README.md

```diff
@@ -1,2 +1,56 @@
+
+## Project pre-requisites
+
+Make sure that you have the compatible versions of [Visual Studio 2022](https://visualstudio.microsoft.com/downloads/) with .NET MAUI workloads and [.NET SDK 7.0](https://dotnet.microsoft.com/en-us/download/dotnet/7.0) or later version in your machine before starting to work on this project.
+
+## How to run this application?
+
+To run this application, you need to first clone the `getting-started-with-the-dotnet-maui-progressbar` repository and then open it in Visual Studio 2022. Now, simply build and run your project to view the output.
+
+## Features and Benefits
+
+### Segments
+Avoid writing several lines of code to create and position multiple progress bars: just split a progress bar into multiple [segments](https://help.syncfusion.com/maui/circularprogressbar/segments?utm_source=github&utm_medium=listing&utm_campaign=maui-progressbar-github-samples) to visualize the progress of multiple sequential tasks.
+
+### Custom content
+Add any view to the center of a circular progress bar as a [content](https://help.syncfusion.com/maui/circularprogressbar/custom-content?utm_source=github&utm_medium=listing&utm_campaign=maui-progressbar-github-samples) to: indicate the completion of a task or process; add start, pause, or cancel buttons to control the progress interactively; add an image that indicates the actual task in progress; and add custom text that conveys the progress unit.
+
+### Visualize multiple ranges
+Along with visualizing the progression of a task, users can also visualize multiple ranges with gradient colors or solid colors based on the range to enhance readability.
+
+## Related Links
+
+[Learn More about .NET MAUI ProgressBar](https://www.syncfusion.com/maui-controls/maui-progressbar?utm_source=github&utm_medium=listing&utm_campaign=maui-progressbar-github-samples)
+
+[Download Free Trial](https://www.syncfusion.com/downloads/maui?utm_source=github&utm_medium=listing&utm_campaign=maui-progressbar-github-samples) 
+
+[Pricing](https://www.syncfusion.com/sales/teamlicense?utm_source=github&utm_medium=listing&utm_campaign=maui-progressbar-github-samples) 
+
+[Documentation](https://help.syncfusion.com/maui/circularprogressbar/getting-started?utm_source=github&utm_medium=listing&utm_campaign=maui-progressbar-github-samples) 
+
+[View Demos](https://github.com/syncfusion/maui-demos/tree/master/MAUI/ProgressBar?utm_source=github&utm_medium=listing&utm_campaign=maui-progressbar-github-samples) 
+
+[Community Forums](https://www.syncfusion.com/forums/maui?control=sfcircularprogressbar?utm_source=github&utm_medium=listing&utm_campaign=maui-progressbar-github-samples)
```

Comment:
Is this link correct?

Reply:
Yes

Reply:
Recheck on the exact link we do use here...

Reply:
Corrected